src/Drive.php

```diff
@@ -277,6 +294,10 @@ public function uploadFile(string $path, string $content): bool
      * @param resource|string|null $resource file resource pointing to the file to be uploaded
      *
      * @return bool
+     * @throws BadRequestException
+     * @throws ForbiddenException
+     * @throws NotFoundException
+     * @throws UnauthorizedException
      * @throws \Exception
```

Comment:
Do we still need lines like this?
Are there other "unusual" exceptions that are known to be possible?

Reply:
we throw the \Exception ourself in this case in line 308. I could change it to `InvalidParameterException`